composer.json

```
@@ -21,14 +21,15 @@
         "php-http/discovery": "^1.0",
         "php-http/httplug": "^1.1 || ^2.0",
         "php-http/message": "^1.6",
-        "php-http/message-factory": "^1.0",
         "php-http/promise": "^1.0",
+        "psr/http-factory": "^1.0",
         "psr/http-message": "^1.0",
         "ramsey/uuid": "^3.7 || ^4.0"
     },
     "require-dev": {
         "ergebnis/composer-normalize": "^2.4",
-        "lmc/coding-standard": "^1.3 || ^2.0",
+        "http-interop/http-factory-guzzle": "^1.2",
```

Shouldn't be required oldest possible version with needed features?
```suggestion
        "http-interop/http-factory-guzzle": "^1.0",
```
I'm affraid that `^1.2` could be problem if some projects use `1.1.x` or `1.0.x` versions.